packages/cli/src/credentials/credentials.controller.ee.ts

```diff
@@ -6,13 +6,15 @@ import express from 'express';
 import * as config from '../../config';
 import { CredentialRequest } from '../requests';
 import { UserService } from '../user/user.service';
-import { EECreditentialsService as EECredentials } from './credentials.service.ee';
+import { EE as Enterprise } from './credentials.service.ee';
```

Comment:
Temp alias needed since `EE` is taken, but only in this spot. Small price for better readability everywhere else.